client/src/app/core/core-services/time-travel.service.ts

```diff
@@ -26,70 +13,25 @@ interface HistoryData {
     providedIn: 'root'
 })
 export class TimeTravelService {
-    public constructor(
-        private httpService: HttpService,
-        private modelMapperService: CollectionMapperService,
-        private DS: DataStoreService,
-        private historyService: HistoryService,
-        private lifecycleService: LifecycleService,
-        private DSUpdateManager: DataStoreUpdateManagerService
-    ) {}
+    public constructor(private historyService: HistoryService) {}
 
     /**
      * Main entry point to set OpenSlides to another history point.
      *
-     * @param history the desired point in the history of OpenSlides
+     * @param position the desired point in the history of OpenSlides
      */
-    public async loadHistoryPoint(history: History): Promise<void> {
-        const updateSlot = await this.DSUpdateManager.getNewUpdateSlot(this.DS);
-
-        await this.stopTime(history);
-        const historyData: HistoryData = await this.getHistoryData(history);
-
-        const allModels = [];
-        Object.keys(historyData).forEach(collection => {
-            const targetClass = this.modelMapperService.getModelConstructor(collection);
-            historyData[collection].forEach(model => {
-                allModels.push(new targetClass(model));
-            });
-        });
-        // await this.DS.set(allModels, 0);
-
-        this.DSUpdateManager.commit(updateSlot);
+    public async loadHistoryPoint(position: Position): Promise<void> {
+        await this.stopTime(position);
     }
 
     /**
-     * Leaves the history mode. Just restart OpenSlides:
-     * The active user is checked, a new WS connection established and
-     * all missed auto updates are requested.
+     * Leaves the history mode.
      */
     public async resumeTime(): Promise<void> {
         this.historyService.leaveHistoryMode();
-        await this.DS.set();
-        await this.lifecycleService.reboot();
     }
```

Comment:
I think, these two functions can be a "synchronous" function (so remove the `async`-keyword) or is there a benefit, when they are asynchronous?